### ISSUE TITLE
SGB Pallet/Border Support?

### ISSUE THREAD

**Issue by Zeldaboy14** [lambertjamesd/gb64#23]
Would it be possible to add a way to toggle the border and pallets for SGB titles that make use of it? Would be very nice to have this sorta feature!

**Comment by lambertjamesd** [comment]
I was planning to at least have some pre packaged border in the major version. I'll see how much effort it would take to implement SGB features but I probably wont implement all SGB features since it would require emulating the super nintendo sound chip and possibly the CPU to do correctly and I don't want to commit to that much effort. If implementing partial features is viable I will consider it.

**Comment by Zeldaboy14** [comment]
Yeah. Borders, and possible the pallets are the only 2 i can see being supported. Trying to do things like what Donkey Kong and one other title dude (using the snes hardware itself) is virtually impossible

**Comment by KevDoy** [comment]
Loving your project. I was also hoping to see SGB border support. Don't much care for emulating the rest of the features. But if there's a simple way to pull the border file out and apply that around the game (maybe as one of the zoom options), I'd love to see it.
